.github/workflows/passport-sdk-sample-app-deployment.yaml

```diff
@@ -33,8 +33,8 @@ jobs:
       - name: setup
         uses: ./.github/actions/setup
 
-      - name: Export Passport sample app
-        run: yarn nx run @imtbl/passport-sdk-sample-app:export --skip-nx-cache
+      - name: Build Static Passport sample app
+        run: yarn nx run @imtbl/passport-sdk-sample-app:build --skip-nx-cache
```

Comment:
Why are we changing this?

Reply:
Nextjs 14 changes the way static export builds are handled. Now we run next build, since the next export command isn't functional anymore, and we specify the output type in the next.config.js file.

Reply:
Sure, should we remove the `export` script then if it isn't functional?

Reply:
Yeap I've removed that in a follow up commit below this comment